pwiz_tools/Skyline/Alerts/MessageDlg.cs

```diff
@@ -29,20 +29,30 @@ public static void Show(IWin32Window parent, string message, bool ignoreModeUI =
             new MessageDlg(message, ignoreModeUI).ShowAndDispose(parent);
         }
 
+        public static void Show(IWin32Window parent, string message, bool ignoreModeUI, MessageIcon messageIcon)

```

Comment:
Changes to MessageDlg and AlertDlg need review by @nickshulman or @brendanx67 I think.